### PR TITLE
Let escape fields with dot (#774)

### PR DESCRIPTION
**Pull request by esaracco**
@NicolasCARPi This is just a proposal. It might need some tweaking. In all cases, it is necessary to test thoroughly. Once everything is OK I will add some tests.
Let me know!

### CODE REVIEW

File: lib/core/parsers/api_param.js [hunk 1]
```diff
@@ -111,19 +111,23 @@ function parse (content, source, defaultGroup) {
   group = matches[1] || defaultGroup || 'Parameter';
 
   const type = matches[2];
-  // accept both '.' and '[]' syntax for object properties
+  // accept both '.' and '[]' for object properties (backward compatibility)
```
[the remark this answers is not in the document]
Comment (esaracco) [reply lib/core/parsers/api_param.js]: Dîne

**Comment by esaracco** [comment]
@NicolasCARPi I'm done with this PR. Let me know if its OK for you or if it needs some rework.

**Comment by NicolasCARPi** [comment]
So what do we do with this PR? Merge, not merge?

**Comment by rottmann** [comment]
@NicolasCARPi At the moment I see nothing that speaks against it. Could be merged.

**Comment by ahsantariq21** [comment]
Can someone tell when we can expect this change on NPM?

**Comment by NicolasCARPi** [comment]
@ahsantariq21 I'll try and polish a bit more the next release that will be 1.0.0. In the meantime, feel free to target `#master` when installing from npm.

**Pull request merged**